fragments/devcontainer/.devcontainer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "tanzu-devcontainer",
+    "image":"projects.registry.vmware.com/tanzu-developer-containers/tanzu-devcontainer-java:latest",
+    "features": {
+        "ghcr.io/devcontainers/features/java:1": {
+            "version": "17"
+        },
+        "ghcr.io/devcontainers/features/docker-outside-of-docker:1": {}
```

Comment:
question: what is this for?

Reply:
No idea really. Maybe something that allows running `docker` commands? @suarezjulian can clarify ?

Reply:
This is needed for `docker login` support to able to push your source code image to a registry

Reply:
For <= tap 1.5?  i.e. versions prior to the local source proxy?

Reply:
Right, this is needed when not using local source proxy, so we can remove if we decide to only support LSP supported clusters